GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
+  - Implementation of the task would conflict with the project direction or [its values](https://biomejs.dev/internals/philosophy/).
+  - Implementation of the task is (currently) technically infeasible.
+  - The suggested pledge is deemed inappropriate for the amount of work involved.
+- The person making the pledge acknowledges their pledge does not come with a guarantee of work done.
+- A task is only completed when a Biome maintainer merges the pull request that closes the task. The person trying to complete the task is responsible for addressing review comments to make sure the work meets the quality standards of the Biome project.
+- There is a 6 month expiration on pledges, as managed by Polar.sh.
+- Terms and conditions of Polar.sh apply to all pledges.
+  - For the FAQ, please see: https://docs.polar.sh/maintainers/faq/
+  - For the legal terms, please see: https://polar.sh/legal/terms
+- If you are receiving money as part of a bounty, please consider whether you need to pay taxes in your country.
+
+### Project-Funded Bounties
+
+In some cases, Biome funds bounties on its own issues. For project-funded bounties, the same conditions apply as for community-funded bounties. But given that these issues are funded by money from our sponsors, we have additional restrictions in place:
+
+- Implementation of the task funded by Biome must advance [our roadmap](#roadmap) or help with essential infrastructure.
+  - The description of the task must include a clear rationale for why this task is eligible for funding.
+- Each Biome-funded task will be either assigned directly to a contributor, or assigned a label to indicate who is eligible for picking up that task:
+  - `F-Anyone`: Anyone is free to pick up this task. This task should be of low complexity and has a maximum bounty of $50 (USD).
+  - `F-Maintainers`: Any Biome maintainer is free to pick up this task. This task may be of medium complexity and has a maximum bounty of $200 (USD).
+  - `F-Core-Contributors`: Only core contributors are eligible for picking this task. There is no upper limit to the complexity, but the maximum bounty is $1000 (USD).
+- For Biome-funded tasks assigned to core contributors, we ask the contributor to take the money directly from our Open Collective account, instead of using Polar.sh. This will help us avoid paying commission over the larger bounties.
+
+Additionally, there is a strict process for assigning project-funded bounties:
+
+- To request Biome funding for a task, a request with a clear rationale must be made in the `Community > #bounties` channel on Discord.
+  - Merely commenting on a GitHub issue is not sufficient, to make sure project members have a better chance at monitoring requests.
```

Comment:
```suggestion
  - Commenting on a GitHub issue might not guarantee enough visibility to ensure project members have a better chance at monitoring requests.
```

I don't know what this bullet point means; I rephrase it in a way where commenting is still a legitimate approach, but it doesn't guarantee the desired outcome.

Reply:
The point here is that Core contributors should only need to watch a single channel without needing to worry they might miss funding-related decisions being made outside that channel. I think this is less of a concern for community-funded issues, since it isn't Biome's money being spent, but it's a relevant concern for issues funded by Biome itself.

So I don't agree with the suggestion here, because it leaves open the possibility that such decisions get made in an issue thread instead. But I guess I may need to reword a bit to make the intention more clear.